manifests/n/Novum/vkQuake/1.30.1/Novum.vkQuake.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Novum.vkQuake
+PackageVersion: 1.30.1
+InstallerType: nullsoft
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 694.)